File: app/src/main/java/com/example/dgtimer/utils/TimeUtils.java
package com.example.dgtimer.utils;

public class TimeUtils {
    public static int stageToSecond(int stage){
        int second = 0;

        switch (stage){
            case 1:
                second = 8;
                break;
            case 2:
                second = 10;
                break;
            case 3:
                second = 15;
                break;
            case 4:
                second = 21;
                break;
            case 5:
                second = 24;
                break;
            case 6:
                second = 28;
                break;
            case 7:
                second = 38;
                break;
        }

        return second;
    }
}

File: app/src/main/java/com/example/dgtimer/utils/TimeUtils.kt
package com.example.dgtimer.utils

object TimeUtils {
    const val MILLIS = 1000L

    fun stageToMilliseconds(stage: Int): Long =
        when (stage) {
            1 -> 8 * MILLIS
            2 -> 10 * MILLIS
            3 -> 15 * MILLIS
            4 -> 21 * MILLIS
            5 -> 24 * MILLIS
            6 -> 28 * MILLIS
            7 -> 38 * MILLIS
            8 -> 60 * MILLIS
            else -> 0L
        }
}